src/main/menu/AppMenu.ts

```diff
@@ -50,7 +47,36 @@ export class AppMenu {
       // File menu
       {
         label: 'File',
-        submenu: [this.isMac ? { role: 'close' as const } : { role: 'quit' as const }],
+        submenu: [
+          {
+            label: 'New Tab',
+            accelerator: this.isMac ? 'Cmd+T' : 'Ctrl+T',
+            click: () => {
+              this.tabManager.createTab();
+            },
+          },
+          {
+            label: 'New Window',
+            accelerator: this.isMac ? 'Cmd+N' : 'Ctrl+N',
+            click: () => {
+              // For now, create a new tab. In future, this could open a new window
+              this.tabManager.createTab();
+            },
+          },
+          { type: 'separator' as const },
+          {
+            label: 'Close Tab',
+            accelerator: this.isMac ? 'Cmd+W' : 'Ctrl+W',
```

Comment:
Maybe we could do something like this to make the code a bit cleaner. Just an example

```suggestion
            const keys = {
              newTab: isMac ? 'Cmd+T' : 'Ctrl+T',
              newWindow: isMac ? 'Cmd+N' : 'Ctrl+N',
              closeTab: isMac ? 'Cmd+W' : 'Ctrl+W',
            };
            
            submenu: [
  { label: 'New Tab', accelerator: keys.newTab, click: () => this.tabManager.createTab() },
  { label: 'New Window', accelerator: keys.newWindow, click: () => this.tabManager.createTab() },
  ];
```

Reply:
good suggestion